lib/kcerts/ssh.go

```diff
@@ -209,7 +209,7 @@ func GenerateUserSSHCert(ca ssh.Signer, certType uint32, principals []string, tt
 	return priv, cert, nil
 }
 
-func makeKeys() (*rsa.PrivateKey, ssh.PublicKey, error) {
+func MakeKeys() (*rsa.PrivateKey, ssh.PublicKey, error) {
```

Comment:
I'd recommend defaulting to ed25519. Keys are smaller, faster to generate, faster to use, and arguably more secure.

Reply:
I agree but the refactor is large enough that it would warrant its own PR, and I'd like to do it after we get auth server testing up (just to make sure there aren't any weird ssh bugs)

Reply:
FWIW, the CERT currently used by the auth I deployed last week is an ED25519 cert, it worked so far? it at least looked like it.

Reply:
(I agree on a separate PR, BTW)